### PR TITLE
[IOTDB-928] Make ENCODING optional in create time series sentence

### PR DESCRIPTION
**Pull request by yuqi1129**

### CODE REVIEW

**Review by Alima777 — APPROVED** [review]
Nice improvement :D

File: server/src/main/java/org/apache/iotdb/db/qp/strategy/LogicalGenerator.java [hunk 1]
```diff
@@ -1169,9 +1170,15 @@ private PartialPath parseFullPath(FullPathContext ctx) {
   public void enterAttributeClauses(AttributeClausesContext ctx) {
     super.enterAttributeClauses(ctx);
     String dataType = ctx.dataType().getChild(0).getText().toUpperCase();
-    String encoding = ctx.encoding().getChild(0).getText().toUpperCase();
     createTimeSeriesOperator.setDataType(TSDataType.valueOf(dataType));
-    createTimeSeriesOperator.setEncoding(TSEncoding.valueOf(encoding));
+
+    TSEncoding encoding = TSEncoding.PLAIN;
```
Comment (qiaojialin) [inline server/src/main/java/org/apache/iotdb/db/qp/strategy/LogicalGenerator.java:1175]: We could use the default Encoding method for each data type. 

image

Comment (yuqi1129) [reply server/src/main/java/org/apache/iotdb/db/qp/strategy/LogicalGenerator.java:1175]: OK, I will assign a default value according to the data type

**Review by qiaojialin — APPROVED** [review]
Good job Yuqi!

**Pull request merged**